eng/pylintrc

```diff
@@ -7,6 +7,7 @@ reports=no
 ignore-paths=
     azure\\mixedreality\\remoterendering\\_api_version.py,
     azure/mixedreality/remoterendering/_api_version.py,
+    (?:.*[/\\]|^)projects/(models/_models.py|_model_base.py|operations/_operations.py|aio\operations\_operations.py)$,
```

Comment:
Shouldn't the last part be `aio/operations/_operations.py`   (with back slash instead of forward slash)?

Reply:
Since I need your change urgently, I pushed a commit to your branch and made the above change to use backslash (although it's purely esthetic, since it works either way). I also resolved a simple merge conflict.